index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>World Top Languages</title>
+</head>
+<body>
+    <header>
+        <section>
+        <div class="first_nav">
+            <a href="https://web.facebook.com/profile.php?id=100073724910623"><img src="Images/facebook_icon.png" alt="Facebook Icon"></a>
+            <a href="https://twitter.com/AtrilSyed"><img src="Images/twitter_icon.png" alt="Twitter Icon"></a>
+            <ul class="firstnav_links">
+              <li><a href="#">English</a></li>
+              <li><a href="#">Mypage</a></li>
```

Comment:
- [ ] Please check my comment in the video, the one I recorded for you to show you some parts 👇  

https://user-images.githubusercontent.com/75126481/219445287-77f6424d-7c40-4042-b961-6d6d7aff7735.mov

Reply:
Thank you @hiromataba for your time dedicated to me You really gave me best suggestions. I will make sure everything.